src/content/blog/2020-10-10-webpack-5-release.md

```diff
@@ -0,0 +1,1427 @@
+---
+title: Webpack 5 release (2020-10-10)
+sort: -202010100
+contributors:
+    - sokra
+---
+
+webpack 4 was released in February 2018.
+Since then we shipped a lot features without breaking changes.
+We know that people dislike major changes with breaking changes.
+Especially with webpack, which people usually only touch twice a year and the remaining time it "just works".
+But shipping features without breaking changes also has a cost:
+We can't do major API or architectural improvements.
+
+So from time to time there is a point where the difficulties pile up and we are forced to do breaking changes to not mess everything up.
+That's the time for a new major version.
+So webpack 5 contains these architectural improvements and the features that where not possible to implement without them.
```

Comment:
```suggestion
So webpack 5 contains these architectural improvements and the features that were not possible to implement without them.
```

Reply:
Isn't that the past from of `be`?

Reply:
`where` refers to physical or metaphorical location while `were` is a verb, it’s the past tense form of `are`.